packages/ember-runtime/lib/ext/string.js

```diff
@@ -27,8 +27,8 @@ if (Ember.EXTEND_PROTOTYPES === true || Ember.EXTEND_PROTOTYPES.String) {
     @for String
     @private
   */
-  StringPrototype.fmt = function () {
-    return fmt(this, arguments);
+  StringPrototype.fmt = function (...args) {
+    return fmt(this, args);
```

Comment:
It is interesting that

```
function(...args) {
  return fmt(this, args);
}
```

and 

```
function() {
  return fmt(this, ...arguments);
}
```

transpile slightly differently. The former with a for loop, the latter with a slice.